gcp/modules/kiali/main.tf

```diff
@@ -0,0 +1,42 @@
+terraform {
+  backend "gcs" {}
+}
+
+variable "secrets_dir" {}
+variable "charts_dir" {}
+variable "kiali_repository" {}
+variable "kiali_tag" {}
+
+module "prometheus" {
```

Comment:
Thats unfortunate that we can't reuse prometheus that comes with Istio.

I have couple of questions:
* Do we care about persistence here?
* What happens to Kiali in case this instance of Prometheus and all metrics are lost?

Reply:
> Do we care about persistence here?

We don't.

> What happens to Kiali in case this instance of Prometheus and all metrics are lost?

Kali Dashboard will only show data since the last restart of Prometheus.

We discussed this previously and agreed it's not really worth it, as the Kali is mostly for "real-time" monitoring and we have the same data in Stackdriver. (Should this turn out to be an issue, we can add a persistent volume for the Prometheus instance in the future.)